src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    public void getStatistic (String fromFileName, String toFileName) throws FileNotFoundException {
```

Comment:
The method signature declares `throws FileNotFoundException`, but the operations inside the method can throw `IOException`. Consider changing this to `throws IOException` to accurately reflect the exceptions that might be thrown.